Dockerfile.RHEL6.tpl

```diff
@@ -2,7 +2,7 @@ FROM rhel6
 
 MAINTAINER docker@softwarecollections.org
 
-RUN yum update -y && yum install -y yum-utils && \
+RUN yum install -y yum-utils && \
```

Comment:
Would it be possible to add `--setopt=tsflags=nodocs`? It makes the layer use less space.